caregiver_application/web_application/__init__.py

```diff
@@ -3,8 +3,9 @@
 def create_app():
     app = Flask(__name__)
     app.config['SECRET_KEY'] = 'not_so_secret_key'
-
+    # from . import model

```

Comment:
Why to have this comment in the file ?

Reply:
Now it is removed, I was supposed to handle the physiological model software integration before separation.